common/constants/shared.ts

```diff
@@ -14,3 +14,7 @@ export const PLUGIN_ID = 'customImportMap';
 export const PLUGIN_NAME = 'customImportMap';
 export const MAPS_APP_DISPLAY_NAME = 'Maps';
 export const MAPS_APP_ID = 'maps-dashboards';
+
+export enum TOOLTIP_STATE {
+  DISPLAY_FEATURES = 'DISPLAY FEATURES',
+}
```

Comment:
Shall we put it to index.ts file? Do we need capitalize its value with space?

Reply:
Ack